modules/vulnerability-management20-configure-report-delivery-destinations-schedule.adoc

```diff
@@ -9,8 +9,23 @@
 You can optionally configure destinations for vulnerability reports and set up a schedule for delivery.
 
 .Procedure
-. To configure destinations for delivery, in the *Create report* window, under *Configure delivery destinations*, click *Add delivery destination*.
+. To configure destinations for delivery, in the *Create report* window, under *Configure delivery destinations*, you can add a delivery destination and set up a schedule for reporting.
```

Comment:
```suggestion
. To configure destinations for delivery you can add a delivery destination and set up a schedule for reporting by using the *Configure delivery destinations* section in the *Create report* window.
```
Not sure the best way to rewrite but have added a suggestion to get away from using "under"